docs/INSTALL.md

```diff
@@ -9,23 +9,27 @@ General ZAC usage instructions may be found in the [README.md](../README.md) fil
 
 - Java JDK 17 and up (we use the `Temurin` distribution)
 
+For mac users you can run the following command to install the required Java version:
+
+```shell
+brew install oracle-jdk17
+```
+
 ### Gradle build
 
 The software is built using Gradle and for the final step using Maven.
 Both a Gradle and a Maven wrapper are included in the source code, so you do not need to install either Gradle or Maven yourself.
 To build the software use the following command:
 
 ```shell
-./gradlew build
```

Comment:
Did you mean to remove this line?

Reply:
Indeed. Mistake?

Reply:
How did this happen.. will add it back

Reply:
Copied the command for your cli, but did cmd-x instead?